_kbarticles/2025-04-07-opensearch-alerting.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "OpenSearch Alerting"
+date: April 7th, 2025
+excerpt: How to setup alerting for your applications on Cloud.gov using OpenSearch
+---
+
+# OpenSearch Alerting
```

Comment:
Can we get some introduction text just explaining that this is a new feature? And generally what they can use alerts for, e.g.

- Alert on 404s to your application
- Alert on specific message contents in your logs

or whatever examples you think are best